WordPressKit/WordPressComOAuthClient.swift

```diff
@@ -113,8 +111,8 @@ public final class WordPressComOAuthClient: NSObject {
         WordPressComOAuthClient.urlSession()
     }()
 
-    private let webauthnSessionManager: SessionManager = {
-        return WordPressComOAuthClient.sessionManager()
+    private let webAuthnSession: URLSession = {
+        WordPressComOAuthClient.urlSession()
     }()
```

Comment:
Love the implicit return 👨‍🍳 💋 

Looking at the diff made me wonder, do we need to wrap in a closure?

```swift
private let webAuthnSession: URLSession = WordPressComOAuthClient.urlSession()
//
// or, assuming the return type from `urlSession()` is `URLSession`:
private let webAuthnSession = WordPressComOAuthClient.urlSession()
```

Reply:
Probably not. I plan to change this part in the final PR of refactoring this class.